style.css

```diff
@@ -0,0 +1,275 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Poppins;
+  font-size: 10px;
+  font-weight: 300;
+  line-height: 3.5em;
+}
+
+h1 {
+  font-size: 9em;
+  text-align: center;
+}
+
+h2 {
+  color: white;
+  text-shadow: #020819 8px -20px 9px;
+  font-size: 5em;
+  text-align: center;
+  text-transform: uppercase;
+  line-height: 4em;
+}
+
+h3 {
+  font-size: 4.2em;
+  text-align: center;
+  line-height: 1em;
+}
+
+h4 {
+  font-size: 1.5em;
+  letter-spacing: 0.4px;
+  line-height: 1em;
+  margin-bottom: 15px;
+}
+
+h5 {
+  font-size: 1.2em;
+  line-height: 1em;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+header > div {
+  padding: 0 25px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid lightgray;
+}
+
+nav {
+  width: 600px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+}
+
+.blackHeart {
+  width: 20px;
+  margin-right: 1rem;
+}
+
+hr {
+  width: 100%;
+  border-top: 2px solid #f6f4f2;
+}
+
+#search {
```

Comment:
Add a width of 100% so it takes the entire width of the screen - right now it's narrower than all the other elements